sdk/Internal/Duration.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Sdk\Internal;
+
+use OpenTelemetry\Trace as API;
+
+final class Duration extends Time
+{
+    /**
+     * @param int $period Duration period in nanoseconds
+     * @return Duration
+     */
+    public static function of(int $period): Duration
```

Comment:
This method seems a little bit odd, to me. The duration implies two points in time. If we need to convert periods in various units of measure(milli, micro, etc), it will be easier to just use `Time::to`.